From 62081aeabe64ee25dfe90c5fcff8e1268bbd382a Mon Sep 17 00:00:00 2001
From: Mirza Krak <mirza.krak@northern.tech>
Date: Mon, 6 Aug 2018 21:21:09 +0200
Subject: [PATCH 1/2] env: Kconfig: remove defaults for SUNXI

That is ENV_IS_IN_FAT and the default offset. Mender build will set
up these correctly

Signed-off-by: Mirza Krak <mirza.krak@northern.tech>
---
 env/Kconfig | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/env/Kconfig b/env/Kconfig
index a3c6298273..b14a5b1789 100644
--- a/env/Kconfig
+++ b/env/Kconfig
@@ -55,7 +55,6 @@ config ENV_IS_IN_FAT
 	bool "Environment is in a FAT filesystem"
 	depends on !CHAIN_OF_TRUST
 	default y if ARCH_BCM283X
-	default y if ARCH_SUNXI && MMC
 	default y if MMC_OMAP_HS && TI_COMMON_CMD_OPTIONS
 	select FS_FAT
 	select FAT_WRITE
@@ -439,14 +438,6 @@ config ENV_EXT4_FILE

 if ARCH_SUNXI

-config ENV_OFFSET
-	hex "Environment Offset"
-	depends on !ENV_IS_IN_UBI
-	depends on !ENV_IS_NOWHERE
-	default 0x88000 if ARCH_SUNXI
-	help
-	  Offset from the start of the device (or partition)
-
 config ENV_SIZE
 	hex "Environment Size"
 	depends on !ENV_IS_NOWHERE
--
2.18.0

